trade-finance-manager-api/src/v1/helpers/amendment.helpers.test.js

```diff
@@ -507,12 +505,13 @@ describe('sendManualDecisionAmendmentEmail()', () => {
 
 describe('sendFirstTaskEmail()', () => {
   const sendEmailApiSpy = jest.fn(() => Promise.resolve(MOCK_NOTIFY_EMAIL_RESPONSE));
-
   const updateFacilityAmendmentSpy = jest.fn(() => Promise.resolve({}));
+  console.error = jest.fn();
 
   beforeEach(() => {
     sendEmailApiSpy.mockClear();
     updateFacilityAmendmentSpy.mockClear();
+    jest.clearAllMocks();
```

Comment:
It looks to me as if `sendEmailApiSpy` and `updateFacilityAmendmentSpy` are not actually spies. As a result, I think that `jest.clearAllMocks();` should clear everything for us?

Reply:
Apologies, this should have been removed.
I have removed this in a new PR - https://github.com/UK-Export-Finance/dtfs2/pull/3899/files